doc/source/admin/scaling.md

```diff
@@ -553,9 +366,24 @@ as explained in [Job Handler Assignment Methods](#job-handler-assignment-methods
 </job_conf>
 ```
 
-As with statically defined handlers, `run.sh` will start the uWSGI process(es), but you will need to start the webless
-handler processes yourself. This is done on the command line like so (note the addition of the `--attach-to-pool`
-option):
+To let gravity know how many webless handler processes should be started set the number of processes to start in the `gravity:` section of galaxy.yml:
+
+```yaml
+gravity:
+  ...
+  process_count:
```

Comment:
Maybe it'd make more sense to control the process_count within their own key,
so 
```
gravity:
  ...
  job_handler:
    _default_: 
      processes: 4
    special: 2
  workflow_handler:
    processes: 2
  celery:
    concurrency: 2
    extra_args: '--debug'
    
```
I think that would look a little better.

Reply:
`processes` as its own key under `job_handler` or `workflow_handler` doesn't distinguish it as "special" from a pool name (tag). How about:

```gravity:
  ...
  job_handler:
    _default_: 
      processes: 4
    special: 2
  workflow_handler: 2
  celery:
    concurrency: 2
    extra_args: '--debug'
```

Reply:
Although how to specify the name template?

Reply:
For the name template we could do:

```
  job_handler:
    _default_: 
      processes: 4
     name_template: the_template_{i}
    special: 2
  workflow_handler:
    _default_:
      processes: 2
```
But since the pools are the top level keys here we can't really have a construct to say that a handler should join multiple pools (and/or tags). Maybe we should make the first portion of the handler name the top level key ?

```
handlers:
  handler:
    processes: 2
    name_template: {name}{i}
    pools:
      - job-handler
      - workflow-scheduler
  handler1:
    processes: 1
    pools:
      - job-handler.special
  handler2:
    processes: 1
    pools:
      - job-handler
      - job-handler.special
```
would be the equivalent of the old documentation for
> - `handler0` and `handler2` will handle tool executions that are not explicitly mapped to handlers
> - `handler1` and `handler2` will handle tool executions that are mapped to the `special` handler tag

(and handler 1 will only handle special, which we didn't spell out before)

Reply:
I've implemented the last example in https://github.com/galaxyproject/gravity/pull/28/commits/a00df1c671fdc01e3fbc42f64a851a45a37a1870